indra/llui/llcombobox.cpp

```diff
@@ -548,7 +548,7 @@ void LLComboBox::createLineEditor(const LLComboBox::Params& p)
 		LLLineEditor::Params params = p.combo_editor;
 		params.rect(text_entry_rect);
 		params.default_text(LLStringUtil::null);
-		params.max_length.bytes(mMaxChars);
+		params.max_length.chars(mMaxChars);
```

Comment:
This is likely risky and will cause unintended bugs, some viewer messages have strict byte limit server side, so if we set a longer string and put in some unicode it will just be cut. Please, don't do this unless you want to check every use case individually.

Reply:
Right, using the maximum size in bytes is correct here, but I'm still curious about why this change was made.

Reply:
Graphical presets have a very short name length limit. Arguably those should be extended to at least ~50 chars (shouldn't be simulator related so it should be fine to do).